src/v/serde/serde.h

```diff
@@ -203,6 +210,7 @@ inline constexpr auto const is_serde_compatible_v
     || std::is_same_v<T, ss::sstring>
     || std::is_same_v<T, bytes>
     || is_duration_v<T>
+    || is_time_point_v<T>
```

Comment:
This also makes me a little uncomfortable for the same reasons.

Reply:
Since time point is a derived type from duration, think should be covered too.